nitro-protocol/contracts/interfaces/INitroTypes.sol

```diff
@@ -21,6 +21,10 @@ interface INitroTypes {
     struct SignedVariablePart {
         VariablePart variablePart;
         Signature[] sigs;
+    }
+
+    struct RecoveredVariablePart {
+        VariablePart variablePart;
         uint256 signedBy; // bitmask
     }
```

Comment:
This is the change in a nutshell. We still used `SignedVariableParts` off chain, but the adjudicator will transform these into  `RecoveredVariableParts` before passing them to the application. The application can then trust the `signedBy` bitmask and does not need to do any signature recovery.